### PR TITLE
Update Playground readme

### LINKED ISSUE #3050: Add readme.md to playground
Since it is now a standalone package, there should be a readme in packages/playground.

Right now, the instructions on how to build/launch the playground app are buried in the vnext folder. They're under "Building react-native-windows repo" which I guess I didn't find intuitive because I couldn't find them when I went looking.

### PR DESCRIPTION
**Pull request by harinikmsft**
Fixes #3050 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/3491)

### CODE REVIEW

File: packages/playground/README.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
-# playground
+# Playground
 
-This package is not published, and is just used to verify a standalone app
\ No newline at end of file
+Playground is a sample standalone application that shows usage of React Native for Windows vNext with several test scenarios. 
+
+## Launching the app
+1. Make sure your development machine has been set up with all the system requirements and dependencies mentioned [here](https://github.com/microsoft/react-native-windows/blob/master/vnext/docs/GettingStarted.md). Make sure a browser is launched and running.
```
Comment (licanhua) [inline packages/playground/README.md]: change browser to Chrome. currently we only support Chrome

File: packages/playground/README.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
+
+2. Open a command prompt in Administrator mode, Navigate to the root folder and install React Native
```
Comment (licanhua) [inline packages/playground/README.md]: It doesn't need admin permission.

File: packages/playground/README.md [hunk 3]
```diff
@@ -1,3 +1,37 @@
...
+`npm install -g react-native-cli`
+
+3. Install React Native for Windows
+`yarn add rnpm-plugin-windows`
+
```
Comment (licanhua) [inline packages/playground/README.md]: add step `yarn install`

File: packages/playground/README.md (hunk 1 above)
Comment (licanhua) [inline packages/playground/README.md]: It's not only for playground, but It's better to ask user to set Developer Mode  in GettingStarted.md

Comment (harinikmsft) [reply packages/playground/README.md]: That's not true - we are able to run against Edge/Chromium and it works as well.

File: packages/playground/README.md (hunk 2 above)
Comment (harinikmsft) [reply packages/playground/README.md]: A bunch of us found that the react-native run-windows command failed to install and deploy the app if it was not run from an admin prompt. I filed #3518 to track this issue.

File: packages/playground/README.md (hunk 3 above)
Comment (harinikmsft) [reply packages/playground/README.md]: Added now.

File: packages/playground/README.md (hunk 1 above)
Comment (harinikmsft) [reply packages/playground/README.md]: Done.

**Pull request merged**